.github/workflows/plugin-sdk-version-check.yml

```
@@ -0,0 +1,48 @@
+  plugin-sdk-version-check:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+
+      - name: Ensure jq is installed
+        run: sudo apt-get update && sudo apt-get install -y jq
+
+      - name: Merge branches
+        uses: ./.github/actions/merge-branches
```

actionlint (1.7.7)

description is required in metadata of "Merge branches" action at "/home/jailuser/git/.github/actions/merge-branches/action.yml"

(action)